README.md

```diff
@@ -228,6 +228,33 @@ This could look something like this in for example your `resources/views/error/5
 This ID can be searched for in the Sentry interface allowing you to find the error quickly.
 
 
+## Using Laravel 5.6 log channels
+
+To configure Sentry as a log channel add the following config to the `channels` section in `config/logging.php`: 
+
+```php
+'channels' => [
+    // ...
+    'sentry' => [
+        'driver' => 'sentry',
```

Comment:
Maybe also mention somewhere you can configure the `level` and `bubble`?

Reply:
Yes. I did that now 😀 

Let me know if you think that covers it.

Reply:
Looks good 👍